pkgs/shells/zsh/zinit/default.nix

```diff
@@ -23,6 +23,8 @@ stdenvNoCC.mkDerivation rec {
     # Installing backward compatibility layer
     install -m0644 zinit{,-side,-install,-autoload}.zsh "$outdir"
     install -m0755 share/git-process-output.zsh "$outdir"
+    mkdir -p "$outdir/doc"
+    install -m0644 doc/zinit.1 "$outdir/doc/zinit.1"
```

Comment:
Prettier to use `installManPage` in `postInstall`.

Reply:
done